.github/workflows/pr-test-on-comment.yml

```diff
@@ -0,0 +1,17 @@
+name: PR Workflow
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  slashCommandDispatch:
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'approved'
+    steps:
+      - name: Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v2
```

Comment:
This command will make it easier to improve the /deploy command in news mono too!